interfaces/policy/helpers.go

```diff
@@ -66,6 +67,19 @@ func checkID(kind, id string, ids []string, special map[string]string) error {
 	return fmt.Errorf("%s does not match", kind)
 }
 
+func checkOnClassic(c *asserts.OnClassicConstraint) error {
+	if c == nil {
+		return nil
+	}
+	if c.Classic != release.OnClassic {
+		return fmt.Errorf("on-classic mismatch")
+	}
+	if c.Classic && len(c.Distros) != 0 {
+		return checkID("distro", release.ReleaseInfo.ID, c.Distros, nil)
+	}
```

Comment:
In the prior test we check if `c.Classic != release.OnClassic`, but this check is only checking if it is a non-empty list. Should this be checking if the release distribution matches something in c.Distros? If that is planned in a followup branch for 'distro' support, then a TODO comment here would be nice.